### PR TITLE
fix/daily-hadith

### PR DESCRIPTION
**Pull request by atoshmukhamedov**

### CODE REVIEW

File: cron/cron.ts [hunk 1]
```diff
@@ -7,8 +7,9 @@ import schedule from 'node-schedule'
 import customKFunction from '#keyboard/custom'
 import fs from 'fs'
```
Comment (Xayrulloh) [inline cron/cron.ts:8]: Did u use fs

File: utils/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const DAILY_HADITH_KEY = 'dailyHadith'
```
Comment (Xayrulloh) [inline utils/constants.ts:1]: Why we need it

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: cron/cron.ts (hunk 1 above)
Comment (atoshmukhamedov) [reply cron/cron.ts:8]: you've used

File: utils/constants.ts (hunk 2 above)
Comment (atoshmukhamedov) [reply utils/constants.ts:1]: Imagine, I removed it and used plain string.
1. How would you provide that you use the same key everywhere?
2. What would you do if the changed? Would you change the key manually everywhere?

Comment (Xayrulloh) [reply utils/constants.ts:1]: I just asked, relax

**Pull request merged**